.github/workflows/linting_comment.yml

```diff
@@ -11,7 +11,7 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Download lint results
-        uses: dawidd6/action-download-artifact@20319c5641d495c8a52e688b7dc5fada6c3a9fbc # v8
+        uses: dawidd6/action-download-artifact@80620a5d27ce0ae443b965134db88467fc607b43 # v7
```

Comment:
I am so confused as to why this complains, but I think v8 is correct

Reply:
no idea either. Version 7 worked for me locally...